### ISSUE TITLE
Different sections in snapshot viewer for manual snapshots versus automatic snapshots

### ISSUE THREAD

**Issue by aletaziar** [mbnuqw/sidebery#161]
I'd like to be able to differentiate between snapshots I manually create and automatic snapshots. If possible, I would like the snapshots limit to also be optionally split so I can limit the number of automatic snapshots (and auto-delete old automatic snapshots) without limiting my manual snapshots.

**Comment by mbnuqw** [comment]
Yeah, I thought about this, but chose the one common list for simplicity. A bit later I will experiment with some new features for snapshots and see if this can be implemented.
